onnx_transformers/pipelines.py

```diff
@@ -694,6 +704,20 @@ def _forward(self, inputs, return_tensors=False):
         else:
             return predictions.numpy()
 
+    def _create_quantized_graph(self, onnx_opt_model_path):
+        #TODO: add option gpt2 if need
+        opt_options = BertOptimizationOptions('bert')
```

Comment:
check model type explicitly, raise an assert or exception if the model is not bert